src/wp-includes/post.php

```diff
@@ -477,28 +478,28 @@ function create_initial_post_types() {
-				'name'                  => __( 'Navigation Menus', 'gutenberg' ),
-				'singular_name'         => __( 'Navigation Menu', 'gutenberg' ),
-				'menu_name'             => _x( 'Navigation Menus', 'Admin Menu text', 'gutenberg' ),
-				'add_new'               => _x( 'Add New', 'Navigation Menu', 'gutenberg' ),
-				'add_new_item'          => __( 'Add New Navigation Menu', 'gutenberg' ),
-				'new_item'              => __( 'New Navigation Menu', 'gutenberg' ),
-				'edit_item'             => __( 'Edit Navigation Menu', 'gutenberg' ),
-				'view_item'             => __( 'View Navigation Menu', 'gutenberg' ),
-				'all_items'             => __( 'All Navigation Menus', 'gutenberg' ),
-				'search_items'          => __( 'Search Navigation Menus', 'gutenberg' ),
-				'parent_item_colon'     => __( 'Parent Navigation Menu:', 'gutenberg' ),
-				'not_found'             => __( 'No Navigation Menu found.', 'gutenberg' ),
-				'not_found_in_trash'    => __( 'No Navigation Menu found in Trash.', 'gutenberg' ),
-				'archives'              => __( 'Navigation Menu archives', 'gutenberg' ),
-				'insert_into_item'      => __( 'Insert into Navigation Menu', 'gutenberg' ),
-				'uploaded_to_this_item' => __( 'Uploaded to this Navigation Menu', 'gutenberg' ),
+				'name'                  => __( 'Navigation Menus' ),
+				'singular_name'         => __( 'Navigation Menu' ),
+				'menu_name'             => _x( 'Navigation Menus', 'Admin Menu text' ),
+				'add_new'               => _x( 'Add New', 'Navigation Menu' ),
+				'add_new_item'          => __( 'Add New Navigation Menu' ),
+				'new_item'              => __( 'New Navigation Menu' ),
+				'edit_item'             => __( 'Edit Navigation Menu' ),
+				'view_item'             => __( 'View Navigation Menu' ),
+				'all_items'             => __( 'All Navigation Menus' ),
+				'search_items'          => __( 'Search Navigation Menus' ),
+				'parent_item_colon'     => __( 'Parent Navigation Menu:' ),
+				'not_found'             => __( 'No Navigation Menu found.' ),
+				'not_found_in_trash'    => __( 'No Navigation Menu found in Trash.' ),
+				'archives'              => __( 'Navigation Menu archives' ),
+				'insert_into_item'      => __( 'Insert into Navigation Menu' ),
+				'uploaded_to_this_item' => __( 'Uploaded to this Navigation Menu' ),
 				// Some of these are a bit weird, what are they for?
-				'filter_items_list'     => __( 'Filter Navigation Menu list', 'gutenberg' ),
-				'items_list_navigation' => __( 'Navigation Menus list navigation', 'gutenberg' ),
-				'items_list'            => __( 'Navigation Menus list', 'gutenberg' ),
+				'filter_items_list'     => __( 'Filter Navigation Menu list' ),
+				'items_list_navigation' => __( 'Navigation Menus list navigation' ),
+				'items_list'            => __( 'Navigation Menus list' ),
 			),
-			'description'           => __( 'Navigation menus.', 'gutenberg' ),
+			'description'           => __( 'Custom post type for navigation menus.' ),
```

Comment:
Really no reason to mention "custom post type" here.

```suggestion
			'description'           => __( 'Navigation menus.' ),
```

could even just omit it, tbh